### ISSUE TITLE
Conflicting message in host terminal

### ISSUE THREAD

**Issue by abuehrle** [weaveworks/scope#2503]
This is a benign message but why does it appear?  Using the Host terminal (on Master), I get: 

`mesg: ttyname failed: Success`

It sorts itself out, but just thought I'd file this as an fyi.

**Comment by 2opremio** [comment]
@abuehrle what Linux distribution was the host running?

**Comment by abuehrle** [comment]
uname -r says: 

`4.4.0-75-generic`

**Comment by 2opremio** [comment]
That's the kernel version. What distribution was it? Is it a cloud instance? `uname -a` _may_ help

**Comment by abuehrle** [comment]
Yes it's the same instance where the terminal is failing.  You have an invite to that one: 
https://cloud.weave.works/app/cold-hill-34

**Comment by rade** [comment]
I am seeing this in Weave Cloud dev and prod. The error also appears when executing `/bin/bash -l` inside a host terminal there.

This seems be a [known problem with glibc](https://github.com/lxc/lxd/issues/1896). Perhaps we can work around it somehow.
